index.html

```diff
@@ -37,20 +37,20 @@
             </div>
         </nav>
 
-        <h1 class="text-center">Unity8</h1>
+        <h1 class="text-center">Lomiri</h1>
         <p>
           <div class="row">
             <div class="col-sm-6">
               <h2>Why?</h2>
               <div>
                 After Canonical announced that they will stop investing in Convergence, the UBports community stepped forward to pick up the development of the <a href="https://ubuntu-touch.io">Ubuntu Touch operating system</a>.
                 Initially, our first priority has been to make sure that every Ubuntu Touch user could continue to use their device, and the response from the community has been overwhelmingly positive.
-                We are now at a point where we believe that we are ready to explore the possibilities of Unity8 on the desktop.
+                We are now at a point where we believe that we are ready to explore the possibilities of Lomiri on the desktop.
               </div>
               <h2>Where?</h2>
               <div>
                 The development will take place on <a href="https://github.com/ubports/unity8">GitHub</a>.<br />
-                You can also join our telegram groups: <a href="https://t.me/UBports_Unity8">Unity8 development group</a> and <a href="https://t.me/ubports">UBports Supergroup</a>
+                You can also join our telegram groups: <a href="https://t.me/UBports_Lomiri">Lomiri development group</a> and <a href="https://t.me/ubports">UBports Supergroup</a>.
```

Comment:
Is the group really named like this, did you try?

Reply:
>Is the group really named like this, did you try?

Honestly, I'm not sure what's correct. But following the original link (to https://t.me/UBports_Unity8), the page asks "Maybe you meant @UBports_Lomiri?"  So that's what I changed the link.  Please correct as needed.

Reply:
it is correct